src/components/Pins.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Pin = ({image}) => {
+    const DEFAULT_IMAGE = "../../public/img/default-thumb.png"
+    return (
+        <div className= "pins">
+            {image? (<img src={image} alt={"venues across location"} height="50" width="50"></img>):(<img src={DEFAULT_IMAGE} alt={"venues across location"} height="50" width="50"></img>)}
```

Comment:
I believe we can update this to be a self closing img tag, if preferred.

Reply:
Commit 6878c64 closes this issue